packages/example-phone/test/automation/spec/call.js

```diff
@@ -3,7 +3,7 @@ import testUsers from '@ciscospark/test-helper-test-users';
 import pkg from '../../../package';
 
 
-describe(`example-phone`, () => {
+describe.skip(`example-phone call`, () => {
```

Comment:
yep. it's really helpful for gating other jobs, since the plugin name roughly corresponds to which service is broken.